pyproject.toml

```diff
@@ -50,3 +52,12 @@ xl2times = "xl2times.__main__:main"
 filterwarnings = ["ignore::DeprecationWarning", "ignore::UserWarning", "ignore::FutureWarning"]
 # show output, print test coverage report
 addopts = '-s --durations=0 --durations-min=5.0 --tb=native --cov-report term --cov-report html --cov=xl2times --cov=utils'
+
+[tool.poe.tasks]
+# Automation of common dev tasks etc.
+# Run with: `poe <target>`, e,g. `poe lint` or `poe benchmark Ireland`.
+# See https://github.com/nat-n/poethepoet for details.
+benchmark = { cmd = "python utils/run_benchmarks.py benchmarks.yml --verbose --run", help = "Run a single benchmark.  Usage: poe benchmark <benchmark_name>" }
+benchmark_all = { shell = "python utils/run_benchmarks.py benchmarks.yml --verbose | tee out.txt", help = "Run the project", interpreter = "posix" }
+lint = { shell = "git add .pre-commit-config.yaml & pre-commit run", help = "Run pre-commit hooks", interpreter = "posix"}
+test = { cmd = "pytest", help = "Run unit tests with pytest" }
```

Comment:
I find this poe tool quite handy - means you don't have to remember/save or maintain script files for the various incantations. The `poe lint` one alone has saved my fingers much typing time :)

Reply:
Very cool! And I see that you don't have to have poetry to use `poe`: https://poethepoet.natn.io/index.html#usage-without-poetry

Is it possible to remove the `--verbose` flag from the default incantation? My understanding of the docs is that you should still be able to run `poe benchmark --verbose`. You don't really need the verbose logs except in the CI because the benchmarks script stores the output of each benchmark in the corresponding `benchmarks/out/<name>/stdout` file.

Reply:
Yeah handy right? Works cross-platform also without having to worry about line ends and slash directions etc. And yeah seems to work fine just with pip (which isn't too surprising because poetry is just a wrapper around pip).

Sure, I've --verbose removed from the `benchmark` script (left in `benchmarks_all` though so it matches the CI runs)